src/pages/docs/concepts/podsi.mdx

```diff
@@ -0,0 +1,123 @@
+import { Callout, Steps } from 'nextra/components'
+
+# Proof of Data Segment Inclusion (PoDSI)
+
+Data uploaded to web3.storage is aggregated together with data from other users of the service. When an aggregate is big enough, it is stored with multiple Filecoin Storage Providers.
```

Comment:
user thought: what is big enough?
Should we state that SPs have requirements of minimum size they are willing to store and we need to aggregate data together to get there? And say that it is typically between 16 and 32 GB